living_environment/living_environment/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = 'living_environment.apps.Living_environmentAppConfig'
```

Comment:
This line seems excess. Such `Living_environmentAppConfig` isn't present (and the name is badly generated anyways).